planning/behavior_path_lane_change_module/src/interface.cpp

```diff
@@ -122,16 +122,19 @@ BehaviorModuleOutput LaneChangeInterface::plan()
 
 BehaviorModuleOutput LaneChangeInterface::planWaitingApproval()
 {
+  module_type_->updateLaneChangeStatus();
+
   *prev_approved_path_ = getPreviousModuleOutput().path;
-  module_type_->insertStopPoint(
-    module_type_->getLaneChangeStatus().current_lanes, *prev_approved_path_);
 
   BehaviorModuleOutput out;
-  out.path = *prev_approved_path_;
-  out.reference_path = getPreviousModuleOutput().reference_path;
-  out.turn_signal_info = getPreviousModuleOutput().turn_signal_info;
-  out.drivable_area_info = getPreviousModuleOutput().drivable_area_info;
-  out.turn_signal_info = getCurrentTurnSignalInfo(out.path, out.turn_signal_info);
+  out = module_type_->getTerminalLaneChangePath();
+  module_type_->insertStopPoint(module_type_->getLaneChangeStatus().current_lanes, out.path);
+  out.turn_signal_info =
+    getCurrentTurnSignalInfo(out.path, getPreviousModuleOutput().turn_signal_info);
+
+  module_type_->setPreviousModulePaths(
+    getPreviousModuleOutput().reference_path, getPreviousModuleOutput().path);
+  setObjectDebugVisualization();
```

Comment:
These two lines has already been updated in `updateData()`. is there any reason adding them here?

Reply:
Done.
https://github.com/autowarefoundation/autoware.universe/pull/6176/commits/82a6f2801083191c01e597b8f1a83c992df41a3c